jiwonjang/app.js

```diff
@@ -0,0 +1,40 @@
+const dotenv = require("dotenv");
+dotenv.config();
+const http = require("http");
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+
+const { DataSource } = require("typeorm");
+
+const myDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE,
+});
+
+myDataSource.initialize().then(() => {
+  console.log("Data Source has been initialized!");
+});
+
+const app = express();
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan("dev"));
+
+app.get("/ping", (req, res) => {
+  res.json({ message: "pong" });
+});
```

Comment:
```suggestion
app.get("/ping", (req, res) => {
  return res.status(200).json({ message: "pong" });
});
```
단순 health check 용도의 Api 라고 하더라도 백엔드 개발자가 status code까지 핸들링해서 보내줄 수 있도록 해주세요!